features/exploredash/src/main/res/layout/buy_gift_card_description.xml

```diff
@@ -28,7 +40,8 @@
         android:textSize="28sp"
         android:gravity="center"
         android:text="@string/buy_card_header_text"
-        style="@style/Body2.Bold"
+        style="@style/Headline4.Bold"
```

Comment:
We don't actually have Body2.Bold style in the design system. There is no 28sp text style either because Ros did this screen before finishing the typography system, but it's closer to Headline4. We can ask him to fix font sizes on this screen.